docs/guides/postgres/snapshot/instant_backup.md

```diff
@@ -35,7 +38,28 @@ demo    Active  5s
 
 > Note: Yaml files used in this tutorial are stored in [docs/examples/postgres](https://github.com/kubedb/cli/tree/master/docs/examples/postgres) folder in github repository [kubedb/cli](https://github.com/kubedb/cli).
 
-We need an Postgres object in Running phase to perform backup operation.
+#### Prepare Database:
+
+We need an Postgres database running to perform backup operation. If you don't have a Postgres instance running, let's create one.
+
+Here, we are going to create a Postgres database and initialize it from script. For more details on how to initialize Postgres database from script in KubeDB, please visit [here](/docs/guides/postgres/initialization/script_source.md).
```

Comment:
We already have doc guide for `Initialize database with script`. Let's skip this part here, because it's distracting. If, Initialize is needed, you can refer to that initialization doc.